### PR TITLE
Add missing information in the K8s navigator

### PR DESCRIPTION
**Pull request by mangulonr**
Added new entities to group, metrics to sort by and attributes to group by.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by mangulonr** [comment]
@brettj-newrelic I added the latest metrics to the doc but also others which have never added.
Feel free to take a look before asking the docs team to merge it.

**Comment by brettj-newrelic** [comment]
Looks good to me @mangulonr @ally-sassman

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 1]
```diff
@@ -109,12 +109,34 @@ The navigator supports visualizing the following Kubernetes entity types:
         Daemonset
       </td>
       <td>
-        Pods Missing (#), Pods Missing (%), Alert Status, Created At
+        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores),  Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Pods Missing (#), Pods Missing (%), Alert Status, Created At
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:112]:
```suggestion
        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores), Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Pods Missing (#), Pods Missing (%), Alert Status, Created At
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 2]
```diff
@@ -109,12 +109,34 @@ The navigator supports visualizing the following Kubernetes entity types:
...
       <td>
         Namespace, Daemonset
       </td>
     </tr>
+    <tr>
+      <td>
+        Job
+      </td>
+      <td>
+        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores),  Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Alert Status, Created At
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:123]:
```suggestion
        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores), Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Alert Status, Created At
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 3]
```diff
@@ -109,12 +109,34 @@ The navigator supports visualizing the following Kubernetes entity types:
...
+      <td>
+        Namespace, Job
+      </td>
+    </tr>
+    <tr>
+      <td>
+        CronJob
+      </td>
+      <td>
+        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores),  Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Alert Status, Created At
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:134]:
```suggestion
        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores), Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Alert Status, Created At
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 4]
```diff
@@ -126,6 +148,17 @@ The navigator supports visualizing the following Kubernetes entity types:
         All Nodes, Node
       </td>
     </tr>
+    <tr>
+      <td>
+        Container
+      </td>
+      <td>
+        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores),  Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Alert Status
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:156]:
```suggestion
        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores), Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Alert Status
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 5]
```diff
@@ -98,7 +98,7 @@ The navigator supports visualizing the following Kubernetes entity types:
         Statefulset
       </td>
       <td>
-        Pods Missing (#), Pods Missing (%), Alert Status, Created At
+        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores),  Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Pods Missing (#), Pods Missing (%), Alert Status, Created At
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:101]:
```suggestion
        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores), Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Pods Missing (#), Pods Missing (%), Alert Status, Created At
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 6]
```diff
@@ -76,18 +76,18 @@ The navigator supports visualizing the following Kubernetes entity types:
...
+        Namespace, Deployment, StatefulSet, DaemonSet, Job, CronJob, Pod, Node
       </td>
     </tr>
     <tr>
       <td>
         Deployment
       </td>
       <td>
-        CPU Used (cores), Memory Used (bytes), Container Restarts, CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s), Alert Status, Filesystem Used (%), Warning Events, Start Time, Created At
+        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores),  Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s),  Pods Missing (#), Pods Missing (%), Alert Status, Created At
```
Comment (ally-sassman) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:90]:
```suggestion
        CPU Utilization (limit), CPU Utilization (request), CPU Used (cores), Memory Utilization (limit), Memory Utilization (request), Memory Used (bytes), Container Restarts (delta), Container Restarts (cumulative), CPU Throttled (%), Network Rx (bytes/s), Network Tx (bytes/s),  Pods Missing (#), Pods Missing (%), Alert Status, Created At
```

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**